tcl/is.tcl

```diff
@@ -534,10 +266,58 @@ namespace eval qc::is {
         return [expr {[string toupper $bool] in {Y N YES NO TRUE FALSE T F 0 1}}]
     }
 
-    proc decimal {number} {
```

Comment:
If we wanted to handler then we could do something like this to convert them into strings:

```
if { [string first e $number] } {
    set number [qc::exp2string $number]
}
```